gui/modes/PressureAssistMode.qml

```diff
@@ -0,0 +1,12 @@
+import QtQuick 2.0
+import "../controls"
+
+// TODO: Implement mode
```

Comment:
This mode will be quite similar to Command Pressure in terms of presentation - e.g. it will show the same graphs and I think even have the same parameters, just the controller logic is different. OK to leave as-is for now, but generally what are you thoughts on how to achieve that without too much code duplication?

Reply:
For now, I am leaving a placeholder because, honestly, the designs are not ready yet, making it hard to start implementing. So I am focusing on the flows first, leaving the modes as a stub for now. Do you have a suggestion on how to approach this?